.coveragerc

```diff
@@ -80,6 +80,10 @@ omit =
     homeassistant/components/digital_ocean.py
     homeassistant/components/*/digital_ocean.py
 
+    homeassistant/components/danfoss_air/__init__.py
```

Comment:
You can change this to `homeassistant/components/danfoss_air/*` to cover all of them.

Reply:
Fixed.